portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,10 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function getData() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('data-container').innerText = quote;
-  });
+async function getData() {
+  const response = await fetch('/data');
+  const Data = await response.json();
```

Comment:
just curious, why is Data capitalized?

Reply:
I believe one of the examples had it that way, so I just kind of followed their format. No particular reason though.

Reply:
ok, thanks!

typically we reserve capital variables for class/library names. We use camelCase for variable names.